backend/src/routes.ts

```diff
@@ -48,6 +49,13 @@ app.get(
   lumensV2V3.v3CirculatingSupplyHandler,
 );
 
+app.get("/api/v2/dex/24h-payments", dex.get24hPaymentsData);
```

Comment:
we don't need to use `v2` for these new ones. Let's keep em as `/api/dex/...`

the v2/v3 endpoints were added because we made some changes to endpoints that were being used by 3rd parties (ideally we'd like to clean up / deprecate the older versions, but need to look into if they're being used still first). Since these are new endpoints we can consider them independent of the other ones so don't need versions